internal/ecs/launchType.go

```diff
@@ -1,18 +0,0 @@
-package ecs
-
-const (
-	ecsFargateLaunchType = "fargate"
-	ecsEC2LaunchType     = "ec2"
-)
-
-// LaunchType represents the current container AWS Launchtype
-// it can be "fargate" or "ec2".
-type LaunchType string
-
-// NewLaunchType returns the container ECS LaunchType.
-func NewLaunchType(fargate bool) LaunchType {
```

Comment:
thanks. 🎉

Reply:
i had to keep the fargate envar thing to backward compatibility though 😢